mlir/lib/Dialect/Affine/Transforms/SuperVectorize.cpp

```diff
@@ -1185,6 +1185,31 @@ static Value vectorizeOperand(Value operand, VectorizationState &state) {
   return nullptr;
 }
 
+/// Returns true if any vectorized loop IV drives more than one index.
+static bool isIVMappedToMultipleIndices(
+    ArrayRef<Value> indices,
+    const DenseMap<Operation *, unsigned> &loopToVectorDim) {
+  for (auto &kvp : loopToVectorDim) {
+    AffineForOp forOp = cast<AffineForOp>(kvp.first);
+    // Find which indices are invariant w.r.t. this loop IV.
+    auto invariants =
```

Comment:
Nit: expand `auto`. In MLIR, we use `auto` when the type is obvious from the statement-level context (e.g., RHS is a cast) or very long to spell (map iterator above).